docs/guides/building/arm-cross-compile.md

```diff
@@ -0,0 +1,162 @@
+# Cross-Compiling Tauri Applications for ARM-based Devices
+
+This guide explains how to cross-compile your Tauri application for ARM-based devices, such as the Raspberry Pi. Compiling directly on the device may not be feasible due to limited RAM, so we'll explore two methods: manual compilation using Linux or WSL on Windows and automatic cross-compilation using a GitHub Action.
+
+## Manual Compilation
+
+Manual compilation is suitable when you don't need to compile your application frequently and prefer a one-time setup. Follow the steps below:
+
+### Prerequisites
+
+:::warning Important Note
+
+Ubuntu 20.04 WSL has issues with webkit dependencies. It's recommended to use Debian instead.
```

Comment:
Just FYI, before merging i'll test 18.04 and 20.04 on my system because we tend to default to ubuntu everywhere else.

Reply:
I remember trying with Ubuntu 20.04, and I had problems with it. Here: https://discord.com/channels/616186924390023171/1056210277894463571/1057382381268508702

Reply:
Yeah, i remembered that thread too, but iirc it worked for me on ubuntu 1804 a while back and in the context of that thread i meant to try out 20.04 myself too but never got to it :/